### ISSUE TITLE
hardness and blastresistance either not being present or not detected by wawla/hwyla

### ISSUE THREAD

**Issue by Varg95** [Sediments/Compressions#18]
![2017-08-29_12 38 28](https://user-images.githubusercontent.com/23167416/29875507-8d48c9b8-8d9a-11e7-8eb8-0c05d1c20358.png)
as you can see the the type of block isnt shown neither are hardness and blastresistance. this started from version 3.

**Comment by Sediments** [comment]
Yeah, I COMPLETELY changed how my blocks work. I'm gonna have to make comparability stuff for a lot of mods now. 

I'm gonna have to take break for a few days now, this rewrite really wore me down. So expect this in like after 5 days or so.
